.ansible-lint

```diff
@@ -1,2 +1,6 @@
 exclude_paths:
-  - submodules
\ No newline at end of file
+  - submodules
+  - playbooks/roles/ansible-config_template
+  - playbooks/roles/ses
+skip_list:
+  - '204'
```

Comment:
:+1: 

Should probably add a comment above mentioning what this rule is about for clarification "Lines should be no longer than 120 chars"

Reply:
Ok.